src/components/Button.tsx

```diff
@@ -76,15 +76,21 @@ export const Button = ({
     </Box>
   );
 
+  const accessibilityProps = {
+    accessibilityRole: 'button' as 'button',
```

Comment:
This isn't a normal cast.

Reply:
accessibilityRole type is a union, but typescript infers `'button'` as a generic string so I think this is needed

Reply:
yep, exactly that.